gpkit/tests/t_vars.py

```diff
@@ -154,13 +154,13 @@ def test_init(self):
     def test_constraint_creation_units(self):
         v = VectorVariable(2, "v", "m/s")
         c = (v >= 40*gpkit.units("ft/s"))
-        c2 = (v >= [20, 30]*gpkit.units("ft/s"))
+        c2 = (v >= np.array([20, 30])*gpkit.units("ft/s"))
         if gpkit.units:
             self.assertTrue(c.right.units)
-            self.assertTrue(c2.right.units)
+            self.assertTrue(NomialArray(c2.right).units)
```

Comment:
should this casting perhaps happen automatically internally? (so that the RHS of an ArrayConstraint is never a raw array but is instead a NomialArray)